src/PlantBundle/Tests/Controller/PlantControllerTest.php

```diff
@@ -0,0 +1,55 @@
+<?php
```

Comment:
Delete this file, it does not have any code at all. As well as we do not test with phpunit.